quran/tests.py

```diff
@@ -0,0 +1,67 @@
+from django.urls import reverse
+from rest_framework.test import APIClient, APITestCase
+from rest_framework.views import status
+
+from .models import Surah, Ayah
+from .serializers import SurahSerializer, AyahSerializer
+
+
+class GetQuran(APITestCase):
+    '''Checking Quran app API'''
+    client = APIClient()
+
+    def test_get_surahs(self):
```

Comment:
Tbh, I don't feel that we need this test. You are just testing `ListAPIView` correctness, which should be tested in DRF, not here.
You are also testing here a loading to DB. It should be tested in somewhere in models or in fixtures.

Reply:
Sura -  it's single sura.
Surah - means more than one sura. so no need to add `s` to the tail.
https://translate.google.com/?source=gtx_c#view=home&op=translate&sl=auto&tl=en&text=%D1%81%D1%83%D1%80%D0%B0%0A%D1%81%D1%83%D1%80%D1%8B

Reply:
@Jihadik I'm not quite sure about issue here. For me it's too early to separate test_models.py test_drf.py etc. This is test of DRF, if I understood it right, basically [following this tutorial.](https://medium.com/backticks-tildes/lets-build-an-api-with-django-rest-framework-32fcf40231e5)

Reply:
@An4ik Thank you, good question. Asked my friends arabophones, the explanation is not that easy. 
Surah is one of the irregular words when it becomes plural. Although it is female, and we always add “t” to females word to make it plural , the plural of Surah is sowar. 

However, here we try to [use english](https://en.wikipedia.org/wiki/S%C5%ABrah), if it's possible in naming of variables.

Reply:
@govza, in a couple of words: these tests are useless :)

Reply:
@govza, I'm simply saying, just delete these tests. You are trying to test some library functions which is unnecessary.